.github/workflows/stale.yml

```diff
@@ -1,19 +0,0 @@
-name: Mark stale issues and pull requests
-
```

Comment:
Stop github from marking issues `stale` and closing them automatically after n months. This was added by mistake in #385 without knowing its exact behaviour. This has already closed almost all issues on the github.